static/app/views/organizationGroupDetails/groupActivityItem.tsx

```diff
@@ -140,6 +141,35 @@ function GroupActivityItem({activity, orgSlug, projectId, author}: Props) {
               author,
             });
       case GroupActivityType.SET_RESOLVED_IN_COMMIT:
+        const deployedReleases: Array<BaseRelease> = [];
+        for (const release of activity.data.commit?.releases) {
+          if (release.dateReleased !== null) {
+            deployedReleases.push(release);
+          }
+        }
```

Comment:
One problem here is that `CommitWithReleaseSerializer` doesn't actually sort the releases in any specific order, so we'll just get an arbitrary release here. We should probably sort these by `(date_released, date_added)` and choose either the oldest or most recent one.

Not sure what the best option is when we have multiple releases. I think probably the earliest release is best since it's the first time that commit appeared in production. We could potentially include a hoverover like "appeared in release X and Y other releases"